src/modules/itemCounter.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable import/no-cycle */
+// import getShows from '../index.js';
```

Comment:
- [ ] Same here, Please remove this linters disablement line and fix all eslint issues yourself.